src/lib/operators/join_index.cpp

```diff
@@ -226,7 +230,7 @@ std::shared_ptr<const Table> JoinIndex::_on_execute() {
         PerformanceWarning("There are multiple table indexes available, but only the first one is used.");
       }
 
-      const auto& table_index = table_indexes[0];
+      const auto& table_index = table_indexes.front();
```

Comment:
Why?

Reply:
I think it was to make it consistent to `chunk_index = chunk_indexes.front()`.
But since they will be removed this is not very important.
I think the code does the same, which way do you prefer?

Reply:
I think one could argue for readability, which is highly subjective here.
In our code base, we find both `[0]` and `front()`.